cli/cmd/uninstall/uninstall.go

```diff
@@ -0,0 +1,439 @@
+package uninstall
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/hashicorp/consul-k8s/cli/cmd/common"
+	"github.com/hashicorp/consul-k8s/cli/cmd/common/flag"
+	"github.com/hashicorp/consul-k8s/cli/cmd/common/terminal"
+	"helm.sh/helm/v3/pkg/action"
+	helmCLI "helm.sh/helm/v3/pkg/cli"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	FlagAutoApprove    = "auto-approve"
+	DefaultAutoApprove = false
+
+	FlagNamespace        = "namespace"
+	DefaultAllNamespaces = ""
+
+	FlagReleaseName       = "name"
+	DefaultAnyReleaseName = ""
+
+	FlagWipeData    = "wipe-data"
+	DefaultWipeData = false
+
+	FlagSkipWipeData    = "skip-wipe-data"
+	DefaultSkipWipeData = false
+)
+
+type Command struct {
+	*common.BaseCommand
+
+	kubernetes kubernetes.Interface
+
+	set *flag.Sets
+
+	flagNamespace    string
+	flagReleaseName  string
+	flagAutoApprove  bool
+	flagWipeData     bool
+	flagSkipWipeData bool
+
+	flagKubeConfig  string
+	flagKubeContext string
+
+	once sync.Once
+	help string
+}
+
+func (c *Command) init() {
+	c.set = flag.NewSets()
+	{
```

Comment:
does this whole section need to be scoped in a `{ }` ?

Reply:
Nope! Thanks for catching that!